include/LLVMSPIRVOpts.h

```diff
@@ -285,6 +289,8 @@ class TranslatorOpts {
   bool PreserveAuxData = false;
 
   BuiltinFormat SPIRVBuiltinFormat = BuiltinFormat::Function;
+
+  bool IsReport = false;
```

Comment:
Please add a small comment about this field. Thanks

Reply:
This field only purpose is to go around iostream interface to make branch in parser routine and it won't last. Should be removed during removal of iostream implementation.